src/main/java/bio/terra/testrunner/common/utils/KubernetesClientUtils.java

```diff
@@ -58,6 +63,9 @@
   private static CoreV1Api kubernetesClientCoreObject;
   private static AppsV1Api kubernetesClientAppsObject;
 
+  private static Map<String, Map<String, String>> componentVersions =
+      new HashMap<String, Map<String, String>>();
```

Comment:
I think this should be stored on the TestRunner object instead of a utils class object.

Reply:
I reorganized the `importComponentVersions()` inside `getComponentVersions()`. I feel that `componentVersions` should still stay with the utils class.

Let's have a quick sync-up if you disagree.